Scripts/Runtime/Core/Steps/DOTweenAnimationStep.cs

```diff
@@ -19,11 +19,14 @@ public sealed class DOTweenAnimationStep : GameObjectAnimationStep
 
         public override void AddTweenToSequence(Sequence animationSequence)
         {
-            animationSequence.AppendInterval(Delay);
             Sequence sequence = DOTween.Sequence();
             for (int i = 0; i < actions.Length; i++)
             {
                 Tween tween = actions[i].GenerateTween(target, duration);
+                if (i == 0)
```

Comment:
I thought this was not necessary anymore, with the fixes made here:  https://github.com/brunomikoski/Animation-Sequencer/commit/95c9dd0ac86436af7e6bf7d3bd5d63f78e086b54

Reply:
Do you mean the fix in this commit?

https://github.com/brunomikoski/Animation-Sequencer/commit/ee79e77f3ce49d1ac14475b02c88f36210388ba6

I'm not convinced that fix is working 100% as it should, I'll email you about this with further details.

Reply:
For clarity, this fix addresses:

"The Animation Step's Delay was being applied to each child Action individually, this was causing the Actions to execute sequentially rather than simultaneously"

Reply:
Tween.SetDelay from docs:

"Sets a delayed startup for the tween. **In case of Sequences behaves the same as DG.Tweening.TweenSettingsExtensions.PrependInterval(DG.Tweening.Sequence,System.Single)** which means the delay will repeat in case of loops (while with tweens it's ignored after the first loop cycle). Has no effect if the tween has already started.

Sequence.Join from docs:

Inserts the given tween **at the same time position of the last tween, callback or intervale added to the Sequence**. Note that, **in case of a Join after an interval, the insertion time will be the time where the interval starts**, not where it finishes. Has no effect if the Sequence has already started.

My take on this given the above info:

Given that  SetDelay() is called on a Tween and not a Sequence, it will not act like PrependInterval(), meaning when each subsequent Tween is Joined to the sequence, it potentially joins after the delay at the Tween's actual start position rather than at the start of the Interval, but this is undocumented from what I can tell.

So We're seeing something like this:

DELAY1 | TWEEN1 |
               DELAY2 | TWEEN2 |
                              DELAY3 | TWEEN3 |

Rather than:

DELAY1 | TWEEN1 |
DELAY2 | TWEEN2 |
DELAY3 | TWEEN3 |

The fix/workaround in  this PR gives this behaviour:

DELAY1 | TWEEN1 |
             | TWEEN2 |
             | TWEEN3 |

It's possible that this is a bug in DOTWeen, if you investigate further and conclude that then maybe we could open a bug with Demigiant instead.

Niall